test/cmd.js

```diff
@@ -467,6 +506,60 @@ describe('express(1)', function () {
     })
   })
 
+  describe('--no-view', function () {
+    var ctx = setupTestEnvironment(this.fullTitle())
+
+    it('should create basic app with no view engine', function (done) {
+      run(ctx.dir, ['--view', 'none'], function (err, stdout) {
```

Comment:
Ah, yes, I forgot that there was discussion a long time ago to make it validate, but that never got added.